Source/CollisionConnect.cpp

```diff
@@ -0,0 +1,81 @@
+#include "CollisionConnect.hpp"
+
+#include <utility>
+
+CollisionConnectCommand::CollisionConnectCommand(std::vector<CollisionConnectData> collisionConnectData):
+        mCollisionConnectData(std::move(collisionConnectData))
+{
+    redo();
```

Comment:
this should get called by the undo framework so you can bin it

Reply:
binned